composer.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "deployecommerce/module-trojan-order-prevent",
+  "name": "salecto2/module-trojan-order-prevent",
```

Comment:
@kapilpandya2203 Use this composer name at Installation instruction in Readme file.